### PR TITLE
Add solution for 2413-smallest-even-multiple-easy (Java)

### PR DESCRIPTION
**Pull request by kat-kan**
Add solution for 2413-smallest-even-multiple-easy

### CODE REVIEW

**Review by wingkwong — CHANGES_REQUESTED** [review]
1. `<SolutionAuthor name="@kat-kan"/>` should be right above the code block. You may check it locally. 
2. Add Time Complexity & Space Complexity for each solution.

**Comment by kat-kan** [comment]
Following comment from @wingkwong , second commit contains fixes.
Thank you for the code review!

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md [hunk 1]
```diff
@@ -57,28 +57,42 @@ class Solution {
...
+**Time complexity** :
 
-<Tabs>
-<TabItem value="java" label="Java">
-<SolutionAuthor name="@kat-kan"/>
+`O(1)` - as this is very simple algorithm
+
+**Space complexity** :
+
+`O(1)` - the only extra variable used in this solution is `result`
```
Comment (wingkwong) [inline solutions/2400-2499/2413-smallest-even-multiple-easy.md:69]: Same as above.

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md [hunk 2]
```diff
@@ -57,28 +57,42 @@ class Solution {
...
\ No newline at end of file
+</Tabs>
+
+**Time complexity** : 
+
+`O(1)` - as this is very simple algorithm
+
+**Space complexity** : 
+
+`O(1)` - no extra variables used in this solution
```
Comment (wingkwong) [inline solutions/2400-2499/2413-smallest-even-multiple-easy.md:98]: same as above

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md [hunk 3]
```diff
@@ -34,16 +34,16 @@ Explanation: The smallest multiple of both 6 and 2 is 6. Note that a number is a
 
 ## Approach 1: If-else statement
 
-<Tabs>
-<TabItem value="java" label="Java">
-<SolutionAuthor name="@kat-kan"/>
-
 The smallest multiple of 2 and a number will be either:
 - the number if it is even or
 - the number multiplied by 2.
```
Comment (wingkwong) [inline solutions/2400-2499/2413-smallest-even-multiple-easy.md:39]: Change it to 

````
The smallest multiple of 2 and a number $n$ will be either

- $n$ if $n$ is even 
- $n$ * 2 if $n$ is odd
````

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md [hunk 4]
```diff
@@ -34,16 +34,16 @@ Explanation: The smallest multiple of both 6 and 2 is 6. Note that a number is a
...
 
 In solution we need to first check if the number is even and return the correct result.
 
+<Tabs>
+<TabItem value="java" label="Java">
+<SolutionAuthor name="@kat-kan"/>
+
 ```java
 class Solution {
     public int smallestEvenMultiple(int n) {
```
Comment (wingkwong) [inline solutions/2400-2499/2413-smallest-even-multiple-easy.md:49]: This solution can be more clean. `result` is not necessary at all.

````
class Solution {
    public int smallestEvenMultiple(int n) {
        if (n % 2 == 0) return n;
        return n * 2;
    }
}
````

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md [hunk 5]
```diff
@@ -34,16 +34,16 @@ Explanation: The smallest multiple of both 6 and 2 is 6. Note that a number is a
 
 ## Approach 1: If-else statement
```
Comment (wingkwong) [inline solutions/2400-2499/2413-smallest-even-multiple-easy.md:35]: Call it `n or 2n`

**Review by wingkwong — CHANGES_REQUESTED** [review]

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md [hunk 6]
```diff
@@ -57,28 +57,42 @@ class Solution {
...
 </TabItem>
 </Tabs>
 
-## Approach 2: Ternary operator
+**Time complexity** :
 
-<Tabs>
-<TabItem value="java" label="Java">
-<SolutionAuthor name="@kat-kan"/>
+`O(1)` - as this is very simple algorithm
```
[the remark this answers is not in the document]
Comment (kat-kan) [reply solutions/2400-2499/2413-smallest-even-multiple-easy.md]: done

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md (hunk 1 above)
Comment (kat-kan) [reply solutions/2400-2499/2413-smallest-even-multiple-easy.md]: fixed

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md (hunk 2 above)
Comment (kat-kan) [reply solutions/2400-2499/2413-smallest-even-multiple-easy.md]: fixed

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md (hunk 3 above)
Comment (kat-kan) [reply solutions/2400-2499/2413-smallest-even-multiple-easy.md]: fixed

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md (hunk 4 above)
Comment (kat-kan) [reply solutions/2400-2499/2413-smallest-even-multiple-easy.md:50]: fixed

File: solutions/2400-2499/2413-smallest-even-multiple-easy.md (hunk 5 above)
Comment (kat-kan) [reply solutions/2400-2499/2413-smallest-even-multiple-easy.md:35]: fixed

**Comment by kat-kan** [comment]
Hi @wingkwong thanks again for the review. Hopefully this time the solution fulfills the standards :)

**Review by wingkwong — APPROVED** [review]
Much better. Will cleanup a bit myself.

**Pull request merged**